src/trio/_core/_traps.py

```diff
@@ -151,40 +151,43 @@ def abort_func(raise_cancel):
        task can't be cancelled at this time, and still has to make sure that
        "someone" eventually calls :func:`reschedule`.
 
-       At that point there are again two possibilities. You can simply ignore
-       the cancellation altogether: wait for the operation to complete and
-       then reschedule and continue as normal. (For example, this is what
-       :func:`trio.to_thread.run_sync` does if cancellation is disabled.)
-       The other possibility is that the ``abort_func`` does succeed in
-       cancelling the operation, but for some reason isn't able to report that
-       right away. (Example: on Windows, it's possible to request that an
-       async ("overlapped") I/O operation be cancelled, but this request is
-       *also* asynchronous – you don't find out until later whether the
-       operation was actually cancelled or not.)  To report a delayed
-       cancellation, then you should reschedule the task yourself, and call
-       the ``raise_cancel`` callback passed to ``abort_func`` to raise a
-       :exc:`~trio.Cancelled` (or possibly :exc:`KeyboardInterrupt`) exception
-       into this task. Either of the approaches sketched below can work::
+       At that point there are again two possibilities. You can simply
+       ignore the cancellation altogether: wait for the operation to
+       complete and then reschedule and continue as normal. (For
+       example, this is what :func:`trio.to_thread.run_sync` does if
+       cancellation is disabled.)  The other possibility is that the
+       ``abort_func`` does succeed in cancelling the operation, but
+       for some reason isn't able to report that right away. (Example:
+       on Windows, it's possible to request that an async
+       ("overlapped") I/O operation be cancelled, but this request is
+       *also* asynchronous – you don't find out until later whether
+       the operation was actually cancelled or not.)  To report a
+       delayed cancellation, you should reschedule the task yourself,
+       and cause it to raise the exception ``exc`` that was passed to
+       ``abort_func``. (Currently ``exc`` will always be a
+       `~trio.Cancelled` exception, but we may use this mechanism to
+       raise other exceptions in the future; you should raise whatever
+       you're given.) Either of the approaches sketched below can
+       work::
```

Comment:
is this block of text modified? Or just reformatted? It's hard to see in the diff

Reply:
Modified, specifically the second to last sentance (which turns into two). I copied this over from the old PR which might explain the formatting...